### PR TITLE
Merchant Logo Settings

### PR DESCRIPTION
**Pull request by junhaoyap**
1. Allow merchant to set logo source
2. Allow merchant to set logo tint color
3. Also added our own logo

screen shot 2018-10-18 at 6 25 47 pm
screen shot 2018-10-18 at 6 25 07 pm

### CODE REVIEW

File: Android/.idea/assetWizardSettings.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="WizardSettings">
+    <option name="children">
```
Comment (alcedo) [inline Android/.idea/assetWizardSettings.xml]: dontt include .idea?

File: Android/xfers_sdk/src/main/java/com/xfers/xfers_sdk/Xfers.kt [hunk 2]
```diff
@@ -29,15 +31,31 @@ class Xfers(val context: Context) {
...
 
+        fun setMerchantApiBase(apiBase: String) {
+            XfersConfiguration.setMerchantApiBase(apiBase)
+        }
+
+        fun setMerchantName(name: String) {
+            XfersConfiguration.setMerchantName(name)
+        }
+
+        fun setMerchantLogo(logo: Int) {
```
Comment (alcedo) [inline Android/xfers_sdk/src/main/java/com/xfers/xfers_sdk/Xfers.kt]: integer? looks wrong

File: Android/xfers_sdk/src/main/java/com/xfers/xfers_sdk/Xfers.kt [hunk 3]
```diff
@@ -29,15 +31,31 @@ class Xfers(val context: Context) {
...
+            XfersConfiguration.setMerchantLogo(logo)
+        }
+
+        fun setMerchantLogoTint(tint: Int) {
+            XfersConfiguration.setMerchantLogoTint(tint)
```
Comment (alcedo) [inline Android/xfers_sdk/src/main/java/com/xfers/xfers_sdk/Xfers.kt]: this one integr too?

File: Android/xfers_sdk/src/main/java/com/xfers/xfers_sdk/utils/XfersConfiguration.kt [hunk 4]
```diff
@@ -15,6 +17,12 @@ object XfersConfiguration {
     private var apiBase = ""
     private var currentCountry: Country? = null
 
+    // Merchant Settings
+    private var merchantApiBase = ""
+    private var merchantName = ""
+    private var merchantLogo: Int? = null
```
Comment (alcedo) [inline Android/xfers_sdk/src/main/java/com/xfers/xfers_sdk/utils/XfersConfiguration.kt]: should have better type casting

File: Android/.idea/assetWizardSettings.xml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="WizardSettings">
```
Comment (alcedo) [inline Android/.idea/assetWizardSettings.xml]: Android/.idea/assetWizardSettings.xml `dont include .idea folders?

File: Android/xfers_sdk/src/main/java/com/xfers/xfers_sdk/utils/XfersConfiguration.kt (hunk 4 above)
Comment (junhaoyap) [reply Android/xfers_sdk/src/main/java/com/xfers/xfers_sdk/utils/XfersConfiguration.kt]: I'll modify this into Int with default merchant logo once I get one from Sabrina

Comment (junhaoyap) [reply Android/xfers_sdk/src/main/java/com/xfers/xfers_sdk/utils/XfersConfiguration.kt]: Added this to a card

**Pull request merged**